v3/src/components/tool-shelf/tool-shelf.tsx

```diff
@@ -11,7 +17,24 @@ import CalcIcon from '../../assets/icons/icon-calc.svg'
 import TextIcon from '../../assets/icons/icon-text.svg'
 import PluginsIcon from '../../assets/icons/icon-plug.svg'
 
-export const ToolShelf = () => {
+import './tool-shelf.scss'
+
+const kCalcHeight = 162
```

Comment:
Constants that only pertain to a particular component should live in its handler.